nftables_test.go

```diff
@@ -74,6 +81,32 @@ func ifname(n string) []byte {
 	return b
 }
 
+// openSystemNFTConn returns a netlink connection that tests against
+// the running kernel in a separate network namespace.
+// cleanupSystemNFTConn() must be called from a defer to cleanup
+// created network namespace.
+func openSystemNFTConn(t *testing.T) (*nftables.Conn, netns.NsHandle) {
+	t.Helper()
+	if !*enableSysTests {
+		t.SkipNow()
+	}
+	runtime.LockOSThread()
```

Comment:
Can you please add a comment stating that netns.New calls `unshare(2)` under the covers, which operates on the current thread?

Reply:
Doneskies :]